src/lib/logical_query_plan/export_node.cpp

```diff
@@ -0,0 +1,38 @@
+#include "export_node.hpp"
+
+#include <sstream>
+
+#include "constant_mappings.hpp"
+
+namespace opossum {
+
+ExportNode::ExportNode(const std::string& init_tablename, const std::string& init_filename,
+                       const FileType init_file_type)
+    : BaseNonQueryNode(LQPNodeType::Export),
+      tablename(init_tablename),
+      filename(init_filename),
```

Comment:
table_name / file_name? I think that's more consistent with the rest of the code.